### DISCUSSION TITLE
Some Atom IDE packages not working?

### DISCUSSION THREAD

**Discussion by ThatXliner** [atom-ide-community/atom#401] [Q&A] [answered]
(Not sure if this is supposed to go to https://github.com/atom-community/atom-ide-javascript etc or here. I can transfer this to an issue if you like)

Just downloaded the latest binary...

<img width="1733" alt="Screen Shot 2022-06-25 at 6 08 20 PM" src="https://user-images.githubusercontent.com/66848002/175795269-2196df47-8ae6-4711-b40d-6180f2b3c037.png">

**Comment by aminya** [comment, marked as answer]
Probably related to https://github.com/atom-community/atom-ide-base/pull/92

If someone does the rest of this migration, it would be great:
https://github.com/electron/remote#migrating-from-remote
